src/config.rs

```diff
@@ -141,6 +144,8 @@ impl Config {
             self.basedir.display()
         );
         for entry in WalkDir::new(&self.basedir)
+            .follow_links(true)
+            .same_file_system(true)
```

Comment:
The [docs say](https://docs.rs/walkdir/2.2.8/walkdir/struct.WalkDir.html#method.same_file_system) this only works on Unix and Windows and returns an error otherwise. I'd like to keep supporting other OSes, so can we make this a config option, instead? We can default to `true`, but it would be good to call it out in the docs for macOS (and other non-Unix, non-Windows) users.